documentation/objects/objects.md

```diff
@@ -223,14 +223,59 @@ In this new window there are three vertical tabs:
  A margin can be configured to include context around this bounding box (size in pixels/voxels).
  Alternatively, it is also possible to include the whole image instead of the individual object images.
 
-Some information on processing the exported `h5` tables in Python:
-The file contains two items at the root level, `images` and `table`.
+## HDF5 export format
+
+[HDF5](https://www.hdfgroup.org/) is a flexible cross platform binary data format. Using Python, you can access the data inside the `h5` files using the [h5py](https://www.h5py.org/) library.
+
+The `h5` file contains two items at the root level, `images` and `table`. Using h5py, you can access them like a python dictionary:
+
+```python
+import h5py
+data = h5py.File("path/to/data.h5", "r")
+data.keys()
+# <KeysViewHDF5 ["images", "table"]>
+```
+
 In the `images` group you will find a subgroup for each object (identifiable by `object_id`) which contains two datasets:
 `labeling` (cutout of the segmentation showing the object) and `raw` (cutout of the raw data showing the object).
-The `table` is saved as a [numpy structured array](https://numpy.org/doc/stable/user/basics.rec.html?highlight=structured#module-numpy.doc.structured_arrays) and holds the selected feature values for each object.
-The "columns" are saved as dtypes (you can see all column names in your table: `f["table"].dtype.names`).
-Individual fields can be accessed by name and index: `f["table"]["User Label"][10]` gives you the user label of the 10th object.
 
+```python
+data["images"].keys()
+# <KeysViewHDF5 ["0", "1", "2", "3", ...]>
+```
+Access individual the `raw` and `labeling` images for each object using the key for the object:
+
+```python
+data["images"]["1"].keys()
+# <KeysViewHDF5 ["labeling", "raw"]>
+data["images"]["1"]["labeling"]
+# <HDF5 dataset "labeling": shape (28, 26, 13), type "|u1">
+# stored as a numpy array, access the data using numpy indexing:
+data["images"]["1"]["labeling"][:]
+# image data...
+```
+
+The measurements `table` is saved as a [numpy structured array](https://numpy.org/doc/stable/user/basics.rec.html?highlight=structured#module-numpy.doc.structured_arrays) and holds the selected feature values for each object.
+The "columns" are saved as dtypes (you can see all column names in your table:
```

Comment:
```suggestion
The "columns" are saved as dtypes (you can see all column names in your table):
```

Reply:
Hey this looks good, just a closing brace and this is done :)

Reply:
oops thanks just fixed with 12c9c70!